src/App.js

```diff
@@ -1,24 +1,30 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { NavLink, Route } from 'react-router-dom';
+import AddItem from './components/AddItem';
+import List from './components/List';
+
+// any css was added in index.css
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <div className="nav">
```

Comment:
Looks good to me! Only thing I might change as the app scales is to create a css file per component (instead of all in index.css), and to preface the class names with the component name (e.g. "App-nav" instead of "nav") so that other files that share similar html don't get accidentally overridden

Reply:
We went ahead and left all the css in the index file for the moment until we could discuss a 'style guide' as a group. Personally, I'm a fan of the BEM structure for organizing class names.

@adriennefriend Do we have any sort of style guide or do you have any suggestions?

Reply:
Thank you for asking, @AliLynne! I'm going to tag @segdeha and @stacietaylorcima in to help with this question!

Reply:
You are totally free to discuss and decide on a styling convention that best suits your group. I love [`emotion` styled-components,](https://emotion.sh/docs/styled) personally. But there are a TON of ways you could go about it and I will completely empower you to make the choice as a group! Go, team!

Reply:
Thank you @stacietaylorcima !